### PR TITLE
[Breadcrumbs article] Don’t hardcode font sizes (ever)

### PR DESCRIPTION
**Pull request by tomayac**
Changes proposed in this pull request:

- Replace 16px by 100%. https://twitter.com/tomayac/status/1207254730197614592?s=20

### CODE REVIEW

**Comment by tomayac** [comment]
CC: @argyleink

**Review by argyleink — APPROVED** [review]
thank you! excellent suggestion.

File: src/site/content/en/blog/building-a-breadcrumbs-component/index.md [hunk 1]
```diff
@@ -355,7 +356,7 @@ maintaining the built-in functionality.
   inline-size: 100%;
   block-size: 100%;
   opacity: .01;
-  font-size: 16px; /* fix iOS zoom */
+  font-size: 100% /* Defaults to 16px; fixes iOS zoom */
```
Comment (argyleink) [inline src/site/content/en/blog/building-a-breadcrumbs-component/index.md:359]:
```suggestion
  font-size: min(100%, 16px); /* Defaults to 16px; fixes iOS zoom */
```

thoughts on this? leverage `min()` to create a more adaptive select input size that won't break if someone changes the document font size?

Comment (tomayac) [reply src/site/content/en/blog/building-a-breadcrumbs-component/index.md:359]: Even better! `<3`

**Review by argyleink — APPROVED** [review]

**Comment by tomayac** [comment]
@rachelandrew It looks like a code owner's approval is needed. Thanks in advance!

**Review by rachelandrew — APPROVED** [review]

**Pull request merged**